tests/ha/single_pod_restart/test_server_pod_restart_api.py

```diff
@@ -1436,9 +1429,6 @@ def test_obj_ver_suspension_server_pod_restart(self):
                     "versioning on %s.", self.object_name, self.f_size, self.bucket_name)
         self.version_etag.update({self.bucket_name: []})
         self.version_etag[self.bucket_name].extend(resp[1])
-        self.version_etag.update({"obj_name": self.object_name})
-        self.version_etag.update({"s3_ver": self.s3_ver})
-        self.is_ver = True
```

Comment:
Are these lines not required?

Reply:
> Are these lines not required?

No, since we have updated cleaup function and made it independent for all type of clean up, we don't need to pass specific info.